workspace.json

```diff
@@ -25,6 +25,7 @@
     "sso-client": "libs/sso-client",
     "sso-e2e": "apps/sso-e2e",
     "styles": "libs/styles",
+    "supabase-auth-service": "apps/supabase-auth-service",
```

Comment:
should use a more agnostic name like `auth-service` because we might or might not vendor lock into Supabase as our IdP in the future. 

If you do change the name here you will have to change the folder name, the paths in `project.json` and `serverless.yml` and `pyproject.toml`, or since it isn't too much right now you can just delete and create a new one and copy the code over